src/pybind/mgr/orchestrator/module.py

```diff
@@ -800,7 +791,9 @@ def _apply_osd(self,
         return HandleCommandResult(-errno.EINVAL, stderr=usage)
 
     @_cli_write_command('orch daemon add osd')
-    def _daemon_add_osd(self, svc_arg: Optional[str] = None) -> HandleCommandResult:
+    def _daemon_add_osd(self,
+                        svc_arg: Optional[str] = None,
+                        method: Optional[str] = None) -> HandleCommandResult:
```

Comment:
Let's check the values already in here:

```suggestion
                        method: Optional[OSDMethod] = None) -> HandleCommandResult:
```